### PR TITLE
Add formplayer to all source

### PR DESCRIPTION
**Pull request by benrudolph**
anyone

### CODE REVIEW

File: fab/const.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     'django_celery',
     'django_pillowtop',
     'formsplayer',
+    'formplayer',
```
Comment (czue) [inline fab/const.py:12]: what's the difference between formsplayer and formplayer?

**Review by czue — APPROVED** [review]

**Comment by czue** [comment]
assume it's correct to have both of those, though is weird. :+1: to merge from my side

File: fab/const.py (hunk 1 above)
Comment (benrudolph) [reply fab/const.py:12]: deprecated name for touchforms :/

Comment (czue) [reply fab/const.py:12]: good times

**Pull request merged**